utm.yaml

```diff
@@ -1317,6 +1271,213 @@ components:
+        response_code:
+          type: integer
+          format: int32
+          description: HTTP response code sent/received in response to request.
+
+    ErrorReport:
+      type: object
+      description: A report informing a server of a communication problem.
+      required:
+      - exchange
+      - problem
+      properties:
+        report_id:
+          type: string
+          maxLength: 128
+          description: ID assigned by the server receiving the report.  Not populated
+            when submitting a report.
+        exchange:
+          allOf:
+          - $ref: '#/components/schemas/RequestResponse'
+          description: The request (by this USS) and response associated with the error.
+        problem:
+          type: string
+          description: Human-readable description of the problem with the response.
+
+    ExchangeRecord:
+      type: object
+      description: |-
+        A record of data exchange (USS - USS or USS - DSS).
+      required:
+      - request_source
+      - exchange
+      properties:
+        request_source:
+          type: string
+          description: |-
+            A coded value that indicates the requesting USS: 'LOCAL' (this USS) or 'REMOTE' (peer USS)
+          enum:
+          - LOCAL
+          - REMOTE
+        exchange:
+          allOf:
+          - $ref: '#/components/schemas/RequestResponse'
+          description: |-
+            Time that the response was received or sent
+
+    PositionRecord:
+      type: object
+      description: |-
+        A record of vehicle telemetry information received by this USS (typically for conformance monitoring).
+      required:
+      - time
+      - telemetry
+      properties:
+        time:
+          allOf:
+          - $ref: '#/components/schemas/Time'
+          description: |-
+            Time that this position data was received by the (local) USS
+        telemetry:
+          $ref: '#/components/schemas/VehicleTelemetry'
+
+    OperationRecord:
+      type: object
+      description: |-
+        A record of a position data gathered through the course of a complete operation
+      required:
+      - positions
+      - operation_id
+      properties:
+        positions:
+          type: array
+          items:
+            $ref: '#/components/schemas/PositionRecord'
+          default: []
+        operation_id:
+          description: ID of the Operation associated with 'positions'
+          allOf:
+          - $ref: '#/components/schemas/EntityID'
+
+    PlanningRecord:
+      type: object
+      description: |-
+        A record of a single attempt to (successfully or unsuccessfully) create an Operational Intent.
+      required:
+      - time
+      - ovns
+      - operation_id
+      - problem
+      properties:
+        time:
+          allOf:
+          - $ref: '#/components/schemas/Time'
+          description: Time that this planning event occurred
+        ovns:
```

Comment:
We need a description for this field.  Which OVNs?  Presumably, these are the OVNs the planning USS was aware of when it was planning the (ultimately unsuccessful) operation.  If that is correct, then there can't be `minItems: 1` since a common problem is likely to be that there were no operations when the USS went to plan, but one was added in the mean time.